store/map/index.js

```diff
@@ -197,6 +197,7 @@ export const actions = {
 }
 
 export const getters = {
+  // TODO: check if  all these functions are needed/used
```

Comment:
Is this still a TODO?

Reply:
Yes there are some unused functions  there. I notified @CindyvdVries . I often add TODO's which can  be picked  up later.

Reply:
Yes there are some unused functions  there. I notified @CindyvdVries . I often add TODO's which can  be picked  up later.